src/python/pants/engine/internals/build_files_test.py

```diff
@@ -230,7 +234,19 @@ def create(self, build_patterns=None, parser=None) -> SchedulerSession:
         def symbol_table_singleton() -> SymbolTable:
             return TEST_TABLE
 
-        rules = create_fs_rules() + create_graph_rules(address_mapper) + [symbol_table_singleton]
+        @rule
+        def build_file_imports_behavior() -> BuildFileImportsBehaviorWrapper:
+            return BuildFileImportsBehaviorWrapper(BuildFileImportsBehavior.warn)
```

Comment:
Should this be `error`? Warnings get swallowed. We probably want all BUILD files to be correct in this test.